include/openssl/ssl.h

```diff
@@ -2244,6 +2244,12 @@ extern const char SSL_version_str[];
 
 int ERR_load_SSL_strings(void);
 
+
+typedef unsigned (*DTLS_timer_cb)(SSL *s, unsigned timer_us);
```

Comment:
I think openssl style is typically to explicitly state the type, i.e. "unsigned int", rather than just "unsigned". Here and in numerous other places.